.gitignore

```diff
@@ -3,7 +3,7 @@
 local server.bat
 todo.txt
 build.js
-
+build*
```

Comment:
This is no good. I use github for deployment of KFC (yes, I know it's not a good practice, but it's what I'm doing), so build must be included.